Tasks/PackerBuildV1/Tests/L0CustomManagedTemplate.ts

```diff
@@ -13,7 +13,7 @@ tr.setInput("additionalBuilderParameters", "{}");
 tr.setInput("customTemplateParameters", "{\"client_id\": \"abcdef\", \"drop-location\":\"C:\\\\folder 1\\\\folder-2\"}");
 
 process.env["RELEASE_RELEASENAME"] = "Release-1";
-
+process.env['AGENT_TEMPDIRECTORY'] = '.';
```

Comment:
Can you explain adding this change?

Reply:
When I ran the tests I got an error "##vso[task.issue type=error;source=TaskInternal;]Unhandled: The "path" argument must be of type string. Received undefined".
So to resolve this error, I had to add this in Tests.

Reply:
Can you please share the command you use to run the tests? It's just for me, for future usage.